### ISSUE TITLE
Image Manager doesn't open

### ISSUE THREAD

**Issue by Xerodesign** [2sic/2sxc#742]
When editing content with a Hyperlink type field with "ShowImageManger" enabled, trying to open the image manager using the dropdown on the field results in the page reloading with a JavaScript error.

`TypeError: c.connectBridge is not a function`

The page then stalls until a reload (at which point it no longer attempts to show the image manager).

**Comment by iJungleboy** [comment]
I tried it and for me everything works. Please ensure you have the telerik-package installed, it's required for the old functionality and if you have dnn8 it's not auto-installed any more but it's there in the extensions somewhere.

**Comment by iJungleboy** [comment]
Pls give feedback if this fixed your issue...

**Comment by Xerodesign** [comment]
Thanks for the input, I'll take a look now and let you know ASAP.

**Comment by Xerodesign** [comment]
I reviewed the installation and looking at the extensions page in the host panel, I have the following installed (appears under libraries):

DotNetNuke Telerik Web Components (`DotNetNuke.Telerik.Web`) version 08.00.00

My DNN Platform version is 8.0.0.

Is there anything else I should be looking for to ensure the Telerik controls are installed?

**Comment by iJungleboy** [comment]
that looks correct. so we need to find out if it's the calling page which causes a problem, or the dialog. 

Please open the edit-dialog in a new window (shift-click on the blue in-page button), then add &debug=true to the url and refresh (you have to refresh). 

Now pls start the js debugger to see in which file etc. you get the error - is in in the ui.html stack, or in some aspx page?

**Comment by Xerodesign** [comment]
OK just ran through your instructions... The page the error appears on is this:

`/desktopmodules/tosic_sexycontent/dist/dnn/ui.html?sxcver=8.3.5.19898#zoneId=2&appId=2&tid=89&mid=1470&lang=en-us&langpri=en-us&langs=[]&portalroot=%2F%2Fd2.dev.com%2F&websiteroot=%2F&user[canDesign]=true&user[canDevelop]=true&approot=%2FPortals%2Feis%2F2sxc%2FContent%2F&dialog=edit&mode=edit&items=[{%22Group%22:{%22Guid%22:%22bdef56ed-7f54-4a25-a40b-262a9ae10dfe%22,%22Index%22:0,%22Part%22:%22content%22,%22Add%22:false},%22Title%22:%22Content%22},{%22Group%22:{%22Guid%22:%22bdef56ed-7f54-4a25-a40b-262a9ae10dfe%22,%22Index%22:0,%22Part%22:%22presentation%22,%22Add%22:false},%22Title%22:%22Presentation+Settings%22}]&debug=true`

And the JavaScript error is (from Firefox):

`TypeError: w.connectBridge is not a function`
`/desktopmodules/tosic_sexycontent/dist/sxc-edit/sxc-edit.js?sxcver=8.3.5.19898`
`Line 567`

**Comment by iJungleboy** [comment]
I tried with FF too, it seems to work. Let me explain what should happen, maybe you can corner it more.

The script tries to open a full-screen iframe and open an aspx-page, which will then show the file-picker. the line of code which throws the error is the line which tries to establish communication between the iframe and the original code (because it needs the link to receive the selected file later on). 

I have to assume that the iframe is constructed, but something goes wrong with the page which should load. If you run fiddler or something, do you see a 404 or a 500 error, when it tries to open the dialog?

**Comment by Xerodesign** [comment]
Thanks again for the tips. I did run the request again, this time logging all page changes. One of the requests had the following URL:

`/Default.aspx?tabid=89&error=Could+not+find+a+part+of+the+path+%27d%3a%5c%5cdnn8%5cdesktopmodules%5cadmin%5cradeditorprovider%5ctoolsfile%5cdefault.toolsfile.xml%27.`

Researching that error, I came upon this thread on the DNN website: https://dnntracker.atlassian.net/browse/DNN-8008

The thread states that the RadEditorProvider extension is disabled by default as of DNN build #8.0.0.585.

To solve the issue, I needed to go into `Host`->`Extensions`, click on the `Available Extensions` tab, and install the `RadEditor Manager` provider. After doing this everything started working again.

So you were right in your very first message; part of the Telerik installation was removed by the DNN by default. Thanks for your help!
